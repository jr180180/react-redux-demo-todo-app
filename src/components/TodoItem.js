import React from 'react';

export default ({ onClick, completed, text }) => {
    return (
        <li
        onClick={ () => onClick() }
        style={{textDecoration: completed ? 'line-through' : 'none'}}
        >
            {text}
        </li>
    );
};