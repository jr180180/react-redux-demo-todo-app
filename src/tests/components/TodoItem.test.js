import React from 'react';
import { shallow } from 'enzyme';
import TodoItem from '../../components/TodoItem';

test('should render TodoItem correctly', () => {
    const wrapper = shallow(<TodoItem />);
    expect(wrapper).toMatchSnapshot();
});

