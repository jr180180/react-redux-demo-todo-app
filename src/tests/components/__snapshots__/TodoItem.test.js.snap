// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TodoItem correctly 1`] = `
<li
  onClick={[Function]}
  style={
    Object {
      "textDecoration": "none",
    }
  }
/>
`;
